coalib/results/result_actions/GeneratePatchesAction.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+from pygments.lexers import guess_lexer_for_filename
+from pyprint.ConsolePrinter import ConsolePrinter
+from pathlib import Path
+
+from coalib.results.result_actions.ResultAction import ResultAction
+from coalib.results.result_actions.ShowPatchAction import ShowPatchAction
+from coalib.output.printers.LogPrinter import LogPrinter
+from coalib.parsing.FilterHelper import FilterHelper
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.coala_modes import mode_normal
+from coalib.parsing.DefaultArgParser import default_arg_parser
+
+
+def show_possibilities(console_printer, i, action):
+    console_printer.print('[{:>4}]  {}. Apply patch (\'{}\')'.format('', i,
+                                                                     action))
+
+
+class GBear():
+    def __init__(self, bear: str):
+        self.bear = bear
+        self.name = bear
+
+
+class GeneratePatchesAction(ResultAction):
+    SUCCESS_MESSAGE = 'Patch applied successfully.'
+
+    is_applicable = staticmethod(ShowPatchAction.is_applicable)
+
+    def apply(self,
+              result,
+              original_file_dict,
+              file_diff_dict):
+        """
+        (G)enerate patches
+        """
+
+        section = Section('cli')
+        section.add_or_create_setting(Setting('default_actions',
+                                              '*: ApplyPatchAction'))
+        section.add_or_create_setting(Setting('no_config', True))
+
+        console_printer = ConsolePrinter()
+        possible_options = []
+        log_printer = LogPrinter()
+        bears_app = []
+        args = default_arg_parser().parse_args()
+        args.files = []
+        to_filename = None
+        to_filename = sorted(result.diffs.items())[0][0]
+        language = guess_lexer_for_filename(to_filename, 'Hello').name
+        section.add_or_create_setting(Setting('files', to_filename))
+        path_dir = Path(to_filename)
+        path_dir = path_dir.parts[len(path_dir.parts) - 2]
+        path = to_filename
+        path = path.rstrip(os.sep)
+
+        filtered_bears = FilterHelper.apply_filters(
+            [['language', language]], None)[0]['cli']
+
+        filtered_bears2 = FilterHelper.apply_filters([['language', 'Python']])
+        print(filtered_bears2)
+
+        b = GBear('SpaceConsistencyBear')
+
+        filtered_bears = list(filtered_bears)
+        filtered_bears.insert(0, b)
```

Comment:
@Nosferatul  Its just defined above, not used.
You can do `filtered_bears.insert(0, GBear('SpaceConsistencyBear'))`